proto/beacon/p2p/v1/types.proto

```diff
@@ -95,13 +95,14 @@ message AttestationData {
 
 message ValidatorRecord {
   bytes pubkey = 1;
-  uint64 withdrawal_shard = 2;
-  bytes withdrawal_address = 3;
-  bytes randao_commitment = 4;
+  // TODO(781): The usage of withdrawal_credentials is not defined in spec. Not used in Prysm yet.
+  bytes withdrawal_credentials = 2;
+  bytes randao_commitment = 3;
```

Comment:
Please name with the hash32 convention from #1025. 

This should be randao_commitment_hash32